include/localization.js

```diff
@@ -36,9 +33,7 @@ module.exports = function LocalizationModule(pb) {
      * @param {Object} request The request object
      */
     function Localization(request){
-        
-        //expected to be lowercase and of the form "en-us"
-        this.language = Localization.best(request).toString();
+        this.language = Localization.best(request).toString().toLowerCase().replace('-', '_');
```

Comment:
Yes, I believe the merge on this file was incorrect.  The changes in this file are a reversal of the changes I made for of commit: https://github.com/pencilblue/pencilblue/commit/f7bcb6fee3190332343c2aaaa6dd97cf63e3191c

I think it is safe to make the assumption.  Any bugs that result will be from the change mentioned above and we will address as a bug fix.